src/resources/views/crud/buttons/create_dummy.blade.php

```diff
@@ -0,0 +1,136 @@
+@php
+// Store configs
+$env = \Config::get('backpack.crud.operations.createDummy.environment', ['local', 'testing']);
```

Comment:
`development`, `dev`, `staging`, `qa` are also used as development environment names. There could be more. I think for production there is only two: `production` and `prod`.

Reply:
@pxpm do you suggest to invert the logic? Check if we are on production and if so, do not show the button?

Following your though, it would work, but I think it's a dangerous approach, if any developer uses other name for his production environment _(don't know if this is possible)_ it would show the button there.

I agree on extending this list by default, adding your suggestions, this is also configurable on backpack configs, so by providing such a long list I think we cover 99% of the cases.

Let me know what you think 🙌